src/Components/Application/Application.jsx

```diff
@@ -23,14 +24,15 @@ import {
 } from "./applicationDataHelpers";
 import { reviewReducer } from "./reviewReducer";
 import { connect } from "react-redux";
-import { newReview } from "../../Actions";
+import { newReview, updateNavbar } from "../../Actions";
 import usePromise from "../../Hooks/usePromise";
 import { getReviewAPI } from "../../requests/get";
 import * as UPDATE from "../../requests/update";
 import { ProgramContext } from "../../Contexts/ProgramContext";
 
+//112 = 56*2 = HEADER_HEIGHT*2
 const PageWrapper = styled.div`
-  padding-top: 50px;
+  padding-top: 112px;
```

Comment:
If we are importing the HEADER_HEIGHT variable, we should use it here. I think the travis CI might fail for unused import error?

Reply:
@ahmedhamodi Having trouble using the headerheight inside the styled'div

Reply:
Strange. Not worth too much investigation if it's not working - so just remove the import at the top so travis CI can pass.